feat(vscode): show language server version inside sidebar status item tooltip

Sysix opened the pull request:
Show the version number of oxlint/oxfmt version, when hovering over the sidebar status item.


<img width="196" height="263" alt="grafik" src="https://github.com/user-attachments/assets/ad1a9be4-e280-483a-b741-cdc41e44fae9" />

camc314 commented:
### Merge activity

* **Dec 26, 10:29 PM UTC**: The merge label '0-merge' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc) once it meets the requirements.
* **Dec 26, 10:29 PM UTC**: `camc314` added this pull request to the [Graphite merge queue](https://app.graphite.com/merges?org=oxc-project&repo=oxc).

Sysix committed:
feat(vscode): show language server version inside sidebar status item tooltip (#17360)

Show the version number of oxlint/oxfmt version, when hovering over the sidebar status item.

<img width="196" height="263" alt="grafik" src="https://github.com/user-attachments/assets/ad1a9be4-e280-483a-b741-cdc41e44fae9" />
